src/status.js

```diff
@@ -0,0 +1,8 @@
+import dataBase from './index.js'; // eslint-disable-line
+
+const clear = () => {
+  const filteredBase = dataBase.filter((c) => c.check === false);
+  localStorage.setItem('dataBase', JSON.stringify(filteredBase));
+  window.location.reload();
+};
```

Comment:
- [ OPTIONAL ]Your application is working fine but you don't have to reload the page to get the UI updated since your application has to be interactive so make sure that you create a function to Update the UI.

Reply:
@Owusu-Desmond sir if i turn it off i have to call HTML gen to be shown there 

that's why i reload it

Reply:
sure